src/Options/Trim.php

```diff
@@ -0,0 +1,58 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Onliner\ImgProxy\Options;
+
+use InvalidArgumentException;
+use Onliner\ImgProxy\Support\Color;
+
+class Trim extends Option
+{
+    /**
+     * @var float
+     */
+    private $threshold;
+    /**
+     * @var Color|null
+     */
+    private $color;
+    /**
+     * @var bool|null
+     */
+    private $equalHor;
+    /**
+     * @var bool|null
+     */
+    private $equalVer;
+
+    public function __construct(float $threshold, string $color = null, bool $equalHor = null, bool $equalVer = null)
+    {
+        if ($threshold && $threshold < 0) {
```

Comment:
зачем отдельно проверять на `$threshold`?
```suggestion
        if ($threshold < 0) {
```

Reply:
да, спс